modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -146,6 +154,90 @@
                table-schema->metabase-field-info
                set)})
 
+(def ^:private ^DateTimeFormatter bq-ts-str-formatter
+  "For some crazy reason, the TIMESTAMP String values returned from [[FieldValue]] look like `1577836800.0`, so create
+  a parser for that.  This is basically '<epoch seconds>.<millis>'."
+  (let [^DateTimeFormatterBuilder fb (doto (DateTimeFormatterBuilder.)
+                                       (.appendValue ChronoField/INSTANT_SECONDS 1 19 SignStyle/NEVER)
+                                       (.appendLiteral \.)
+                                       (.appendValue ChronoField/MILLI_OF_SECOND))]
+    (.toFormatter fb)))
```

Comment:
For next time. I wrote a Clojure DSL around `DateTimeFormatterBuilder` that would have made this a little simpler

https://github.com/metabase/metabase/blob/master/src/metabase/util/date_2/parse/builder.clj#L145-L165